### ISSUE TITLE
TensorFlow install issue 

### ISSUE THREAD

**Issue by upavan** [tensorflow/tensorflow#8623]
Hi Team .,

While installing TensorFlow on Windows10 I'm facing the below error., 
>pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/cpu/tensorflow-1.0.1-cp35-cp35m-win_amd64.whl

****tensorflow-1.0.1-cp35-cp35m-win_amd64.whl is not a supported wheel on this platform.****

Please let me know how to overcome this issue.

Thanks
Pavan

**Comment by jdvala** [comment]
Just try this
`pip3 install tensorflow`

this should do the trick

**Comment by upavan** [comment]
>pip3 install tensorflow
**'pip3' is not recognized as an internal or external command,
operable program or batch file.**
Note : My anaconda version is : Anaconda3-4.3.0.1-Windows-x86_64  (python 3.6.0)

Help me how can I resolve this ... Thanks

**Comment by lxsgdsgg** [comment]
are u using pyhton 2.7?
if so , try
pip install tensorflow

**Comment by upavan** [comment]
>pip install tensorflow
Collecting tensorflow
  **Could not find a version that satisfies the requirement tensorflow (from versions: )
No matching distribution found for tensorflow**

**Comment by gunan** [comment]
On windows, we only support python 3.5. We do not have support for python 2.7
This is clearly stated in our installation guide here:
https://www.tensorflow.org/install/install_windows

Closing this issue, as this is not an issue in TF itself.

**Comment by upavan** [comment]
Hi.,
I'm using python 3.6 Version. Still facing the issue.. If you are interested in closing the issue, go ahead and close, but my issue is not resolved.   My environment details : OS - Windows 10 and Python 3.6 and how would you know my python version is 2.7 (no where I mentioned) ?

**Comment by jsondoo** [comment]
@upavan It seems like you're using python 3.6? You can create and activate a new environment with python version 3.5 by doing `conda create --name tensorflow python=3.5`

**Comment by gunan** [comment]
We currently only have support for python 3.5 on windows, as stated in our installation documentation.
I assumed you used python 2.7, as you did not respond negatively to @lxsgdsgg 's query.
Sorry for the misunderstanding.

If you are using anaconda, you can follow the suggestion by @jsondoo to create an environment that has python 3.5

**Comment by upavan** [comment]
Hi Team.,

Thanks.. I guess TensorFlow got installed on my PC.. Thanks a lot.. 

And one more issue with Jupyter/ Spyder :  While working with Jupyter/ Spyder ., I'm facing the below error : **ModuleNotFoundError: No module named 'tensorflow'**  --- Do I need to install any other external software / Packages.. Please let me know how can I work with GUI tools like Jupyter/ Spyder .. meanwhile I'll try with my PC restart.

Thanks 
Pavan

**Comment by gunan** [comment]
Im glad the issue is resolved for you.
For the issue with Jupyter/Spyder, it sounds to me like a setup issue with Jupyter.

Github is a direct channel to tensorflow developers for reporting bugs in tensorflow itself.
It sounds like you are having some problems with Python/Jupyter/Spyder setup, which are standalone pieces of software themselves.
I recommend reaching out to stackoverflow for general help with these software, or looking into product forums to see what is the best way to setup jupyter and set it up to use a specific conda environment you are creating.

**Comment by upavan** [comment]
Thanks for the info.. can you give me the link for stackoverflow of github, I'm trying to post but I didn't find the link., please share me the link to post my issue... thanks

**Comment by gunan** [comment]
stackoverflow is stackoverflow.com
For product forums, you will need to google search the products you are using and browse through the results.

**Comment by upavan** [comment]
Thanks a lot... you are amazing ....

**Comment by davtomic** [comment]
@jsondoo i tried creating conda create --name tensorflow python=3.5 but this error came:
"prefix already exists c:\Anaconda\envs\tensorflow" which makes sense, i dont understand how i can name it tensorflow when tensorflow is already installed via anaconda

**Comment by santhoshshivanna** [comment]
@davtomic  i got the same error. you must first remove the tensorflow environment you had already created. you can do that through command line which i couldnt figure out. So I removed it form anaconda navigator under Environments Tab. After removing you try to create a new environment with python=3.5

**Comment by wiselyloh** [comment]
it works for me.
delete the tensorflow env at anaconda navigator under Environments Tab. then create tf tenv for 3.5 again. then no more error below
tensorflow-1.0.1-cp35-cp35m-win_amd64.whl is not a supported wheel on this platform.

simply, tensorflow not supporting 3.6 yet?

**Comment by RenzhiDuan** [comment]
"conda create -n tensorflow  python=3.5"  also works for me~
I've installed Anaconda 4.3.1 with python 3.6 and following the guidance from 
[https://www.tensorflow.org/install/install_windows#common_installation_problems](url) to install tensorflow with it and meeting the "not supported wheel" problem. 
I just deactivate the former conda environment and remove it with:
"deactivate tensorflow"
"conda env remove -n tensorflow"
And then repeating the guidance with additional "python=3.5" after the creating cmd fixed the problem
 "conda create -n tensorflow  python=3.5" has downloaded and installed python3.5 to this new environment~

**Comment by Geekly** [comment]
I'm having the same problem. Running Anaconda on Windows. I've created an environment and installed python 3.5 it in.
```    
$ python --version
Python 3.5.3 :: Continuum Analytics, Inc.
(py35_64)
```
When I execute 
```
pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/cpu/tensorflow-1.2.1-cp35-cp35m-win_amd64.whl
```
I get the error:
```  
 tensorflow-1.2.1-cp35-cp35m-win_amd64.whl is not a supported wheel on this platform.
```

**Comment by AmiraDeuraseh** [comment]
I got same error with @Geekly 
Using Python 3.5, Windows 10 32-bit.

**Comment by gunan** [comment]
There is no support for 32 bit operating systems. You can try building from sources yourself, but there may be some issues even during build.

**Comment by upavan** [comment]
Try with the latest version of TensorFlow your issue will get resolve., try
on 64Bit version.

**Comment by aritrabasu104** [comment]
Getting the same error as @Geekly.. When I try
`pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/cpu/tensorflow-1.3.0-cp35-cp35m-win_amd64.whl`
however, when i try 
` pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/cpu/tensorflow-1.3.0-**cp36-cp36m**-win_amd64.whl`

everything excep tensorflow gets installed succesfully

**Comment by gunan** [comment]
You should just try running
```
pip install tensorflow
```
Pip will pick the correct version for you based on your OS, python version, etc.

**Comment by prachm** [comment]
I can confirm that `pip install tensorflow` works. My configuration - Windows 7, 64 bit. I upgraded to Anaconda 4.4 and python 3.6.2. I had a previous installation of TensorFlow 1.2 installed using the following command `pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/cpu/tensorflow-1.3.0rc2-cp35-cp35m-win_amd64.whl `

All I had to do was `pip uninstall tensorflow` followed by `pip install tensorflow`
